Gemfile.lock

```diff
@@ -24,6 +24,7 @@ PATH
   specs:
     decidim-admin (0.0.1.alpha5)
       active_link_to (~> 1.0.0)
+      carrierwave (~> 1.0.0.beta)
```

Comment:
Because it's the first one to support Rails 5. I think it's safe enough, but we can discuss this and see if `paperclip` or similars have support for Rails5 in a stable release.